Code/Framework/AzCore/AzCore/Math/ShapeIntersection.h

```diff
@@ -37,13 +37,17 @@ namespace AZ
         bool Overlaps(const Sphere& sphere, const Plane& plane);
         bool Overlaps(const Sphere& sphere1, const Sphere& sphere2);
         bool Overlaps(const Sphere& sphere, const Obb& obb);
+        bool Overlaps(const Sphere& sphere, const Capsule& capsule);
         bool Overlaps(const Frustum& frustum, const Sphere& sphere);
         bool Overlaps(const Frustum& frustum, const Obb& obb);
         bool Overlaps(const Frustum& frustum, const Aabb& aabb);
         bool Overlaps(const Capsule& capsule1, const Capsule& capsule2);
         bool Overlaps(const Capsule& capsule, const Obb& obb);
         bool Overlaps(const Capsule& capsule, const Sphere& sphere);
         bool Overlaps(const Obb& obb1, const Obb& obb2);
+        bool Overlaps(const Obb& obb, const Capsule& capsule);
+        bool Overlaps(const Obb& obb, const Sphere& sphere);
+
```

Comment:
there is a not to verify that all these are symmetric how would you add a test case for that?

Reply:
For a lot of the unit tests I wrote, I called the Overlaps function which both orderings of the arguments. I'll update the rest of the tests to make them call both ways.